### PR TITLE
adding new features to platform status included in the Firefox 63 release

### PR DESCRIPTION
**Pull request by chrisdavidmills**
* Updated CSS motion-path to mention that the properties are now called `offset-path-*`
* Updated Custom Elements to try to stop it saying that there are no signals from Opera. I know for a fact it is enabled by default in Opera/Chrome.
* Added Media Capabilities API
* Added CSS logical properties.

### CODE REVIEW

File: features/css-logical-properties.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: CSS Logical Properties and Values
+category: css, layout
+bugzilla: 1323940
+firefox_status: in-development
+spec_url: https://drafts.csswg.org/css-logical/
+caniuse_ref: css-logical-props
+chrome_ref: 6237096230518784
+ie_ref: csslogicalpropertieslevel1
```
Comment (stof) [inline features/css-logical-properties.md]: broken IE ref. That is the slug, not the name...

File: features/media-capabilities.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Media Capabilities: Decoding
```
Comment (stof) [inline features/media-capabilities.md]: how is that different from the previous one ?

File: features/css-logical-properties.md (hunk 1 above)
Comment (chrisdavidmills) [reply features/css-logical-properties.md]: I think I've fixed this now...

File: features/media-capabilities.md (hunk 2 above)
Comment (chrisdavidmills) [reply features/media-capabilities.md]: Oops, I ended up with a dupe! I've deleted it, and removed the character that I think was making travis fail

Comment (stof) [reply features/media-capabilities.md]: yeah, `: ` is special in YAML, so using it inside a string requires quoting it.

Comment (chrisdavidmills) [reply features/media-capabilities.md]: Good to know, thanks! Looks like we can merge this now then.

**Pull request merged**